mtags/src/main/scala-3/scala/meta/internal/pc/IndexedContext.scala

```diff
@@ -12,6 +12,7 @@ import dotty.tools.dotc.core.Names.*
 import dotty.tools.dotc.core.Symbols.*
 import dotty.tools.dotc.core.Types.*
 import dotty.tools.dotc.typer.ImportInfo
+import dotty.tools.dotc.core.Flags
```

Comment:
Scalafix is failing because of it, I think we can remove this import since it's not used